packages/react-server/src/server/test/floopydoop.test.tsx

```diff
@@ -79,16 +75,14 @@ describe('createServer()', () => {
           ),
         }),
       {
-        headers: {
-          cookie: `${cookie}=${value}`,
+        cookies: {
```

Comment:
✨  cookies also work just fine :)

Reply:
Oh lol I changed this name to make it easier to pattern for with jest but that should probably change back